### PR TITLE
docs(README): fix clone instruction indentation

### PR DESCRIPTION
**Pull request by TejasQ**
A portion of the README that isn't code is marked as code. This is a minor adjustment to that. 😬 

|Main|This PR|
|---|---|
|image|image|

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +29,15 @@ Before you can start working with Yari, you need to:
...
-            git clone https://github.com/[your_account]/content.git
-            git clone https://github.com/[your_account]/yari.git
+```shell
+git clone https://github.com/[your_account]/content.git
+git clone https://github.com/[your_account]/yari.git
+```
 
-        Take a note of the file path to the location where you've cloned that
-        repo before moving on.
+Take a note of the file path to the location where you've cloned that
```
Comment (stof) [inline README.md:37]: I think this was expected to be part of the list item, and so should be indented (but by 4 spaces, not by 8)

Comment (stof) [reply README.md:37]: and the triple-backtick codeblock should also have this 4 space indentation (unlike in your "keep whitespace" attempt)

Comment (TejasQ) [reply README.md:37]: Hmmm, the problem is if we indent it, it becomes a code snippet. 🤔 I'm not really sure how to indent it without this.

Comment (danielhjacobs) [reply README.md:37]: &emsp;You can use \&emsp; to add space approximately equal to four spaces, as in this line.
&nbsp;&nbsp;&nbsp;&nbsp;You can alternatively use \&nbsp; four times in a row, as in this line.

Comment (TejasQ) [reply README.md:37]: Wow, thanks a lot, @danielhjacobs !

Comment (TejasQ) [reply README.md:37]:
```suggestion
 Take a note of the file path to the location where you've cloned that
```

File: README.md [hunk 2]
```diff
@@ -29,13 +29,15 @@ Before you can start working with Yari, you need to:
 1.  Clone the forked repositories to your computer using the following commands
     (replace `[your account]` with the account you forked the repositories to):
 
-            git clone https://github.com/[your_account]/content.git
-            git clone https://github.com/[your_account]/yari.git
+```shell
+git clone https://github.com/[your_account]/content.git
+git clone https://github.com/[your_account]/yari.git
+```
```
Comment (caugner) [inline README.md:35]:
```suggestion
    ```shell
    git clone https://github.com/[your_account]/content.git
    git clone https://github.com/[your_account]/yari.git
    ```
```

File: README.md [hunk 3]
```diff
@@ -29,13 +29,15 @@ Before you can start working with Yari, you need to:
...
 
-        Take a note of the file path to the location where you've cloned that
-        repo before moving on.
+ Take a note of the file path to the location where you've cloned that
+repo before moving on.
```
Comment (caugner) [inline README.md:38]:
```suggestion
    Take a note of the file path to the location where you've cloned that
    repo before moving on.
```

**Review by caugner — CHANGES_REQUESTED** [review]
For me, indenting the code fence and the following line with 4 spaces worked:

**Comment by caugner** [comment]
Sorry about that failing check.

I'm pretty sure that markdown is valid, but I believe we stumbled upon a `markdownlint` bug (or limitation), so I have reported this upstream: https://github.com/DavidAnson/markdownlint/issues/509

**Comment by caugner** [comment]
> I'm pretty sure that markdown is valid, but I believe we stumbled upon a `markdownlint` bug (or limitation)

The markdown was valid and there was _no_ bug in `markdownlint`, but instead it reported that we were using both code fences and intended code blocks. So the solution was to use indentation (4 spaces) in the list item as well.

**Review by caugner — APPROVED** [review]

**Pull request merged**